recipe/meta.yaml

```diff
@@ -28,22 +28,30 @@ build:
 
 requirements:
   build:
-    - {{ compiler('c') }}  # [not win]
+    - {{ compiler('c') }}    # [not win]
     - {{ compiler('cxx') }}  # [not win]
-    - python 3.8  # [not win]
-    - ninja-base  # [not win]
-    - pkg-config  # [not win]
-    - patch     # [not win]
+    - python 3.8             # [not win]
+    - ninja-base             # [not win]
+    - pkg-config             # [not win]
+    - patch                  # [not win]
+    # libuv is a bootstrap dependency in js2c
+    # starting with 20.3 (https://github.com/nodejs/node/pull/46997)
+    - libuv
   host:
-    - libuv 1.44    # [not win]
-    - openssl {{ openssl }} # [not win]
-    - zlib 1.2      # [not win]
-    - icu 73
+    - icu {{ icu }}
+    # The introduction of the uv_pipe_bind2 and uv_pipe_connect2 methods in
+    # libuv v1.46.0 changed the behaviour of uv_pipe_bind and uv_pipe_connect.
+    # see https://github.com/nodejs/node/pull/49667 and https://docs.libuv.org/en/v1.x/pipe.html#c.uv_pipe_bind2
+    # uv_pipe_bind2 added in v20.8.0 of the 20.x branch
+    # https://github.com/nodejs/node/blob/v20.8.0/src/pipe_wrap.cc
+    - libuv 1.48             # [not win]
+    - openssl {{ openssl }}  # [not win]
+    - zlib {{ zlib }}        # [not win]
   run:
     - {{ pin_compatible('icu') }}
-    - libuv  # [not win]
-    - openssl # [not win]
-    - zlib  # [not win]
+    - libuv                  # [not win]
+    - openssl                # [not win]
+    - zlib                   # [not win]
   run_constrained:   # [osx]
     - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.9") }}   # [osx and x86_64]
     - __osx >={{ MACOSX_DEPLOYMENT_TARGET|default("10.15") }}  # [osx and arm64]
```

Comment:
Can you align the default value with what is in the cbc?

Reply:
Done